### DISCUSSION TITLE
Query with "BETWEEN"

### DISCUSSION THREAD

**Discussion by ROiPinheiro** [prisma/prisma#2045] [Q&A] [answered]
how do I make one query like this with prisma ??

```sql
SELECT * FROM table WHERE 1000 BETWEEN startNum AND endNum;
```

**Comment by Sytten** [comment, marked as answer]
On the top of my head I would inverse the condition and do this:
```javascript
prisma.table.findMany({
    where: { AND: [{ startNum: { lte: 1000 } }, { endNum: { gte: 1000 } }] },
  });
```

**Comment by timsuchanek** [comment]
Correct! You can even just write this, as AND is the default behavior:
```ts
prisma.table.findMany({
  where: { startNum: { lte: 1000 }, endNum: { gte: 1000 } },
});
```

**Reply by Sytten** [reply]
Thats neat I didnt know that, is it documented?

**Reply by pantharshit00** [reply]
It is not documented. Opened: https://github.com/prisma/prisma/issues/2126

**Comment by ethanseow** [comment]
For anyone who finds this, the AND positioning does matter. Connection:{lte:67}, Connection:{gte:51} results in something else if the order is flipped (gte then lte)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
